app/views/instruments/schedule.html.haml

```diff
@@ -40,7 +40,7 @@
   %p.notice= t(".no_upcoming_admin_reservations")
 
 - else
-  - unless @instrument.online?
+  - if @instrument.offline?
```

Comment:
I could see this one going either way, but I lean towards having the icon for each of the items in the shared calendar.